liquidity-module/index.md

```diff
@@ -23,28 +28,38 @@ In this tutorial, you create your own blockchain, send tokens to another blockch
 - Create a pool with your token
 - Use the pool with your token
 
-## Prerequisites
+
+## Requirements
 
 Before you start the tutorial, install the prerequisite software. 
 
-- [Install Starport](../starport/index.md) v0.16.2 <!-- link to the new tutorial file for this prereq to install Starport https://github.com/cosmos/sdk-tutorials/pull/694/commits/9a988d64408df16dad61412b7c542f6dd1fa4bee  -->
+- [Install Starport](../starport/index.md) v0.16.2
 
     **Important** This tutorial uses [Starport](https://github.com/tendermint/starport) v0.16.2. The tutorial is based on this specific version of Starport and is not supported for other versions.
 
-- Install the Gravity DEX binary:
+- Install the Gravity DEX binary
+
+    - Clone the repo:
 
     ```bash
     git clone https://github.com/b-harvest/gravity-dex.git
-    cd gravity-dex
-    make install
+    ```
+    
+    - Copy the software to the right location:  
 
+    ```bash
+    cd gravity-dex && make install
+    ```
+    - Verify the gaiad version:  
+
+    ```bash
     gaiad version
     ```
 
-    The output of `gaiad version` should print something like
+    The output of `gaiad version` prints something like:
     
     ```bash
-    gravity-dex-fa647b0fefe5508e9c975b3d4f095db2d3d20a13
+    add-liquidity-module-to-gaia-022f6eea5ec93995bc175c556e37e30cae22e1f6
```

Comment:
Hi Barrie, thanks for catching the wording.

`gravity-dex-fa647b0fefe5508e9c975b3d4f095db2d3d20a13` is literally the version tag of this build. Therefore it is the expected output when running `gaiad version`.